SpeechModel.js

```diff
@@ -1,19 +1,13 @@
-var speechModel;
-
 class SpeechModel {
 
 	constructor() {
-		speechModel = this;
-		// for training
-		this.iteration = 5;
-
 		// this config is place holder (copied from CNN_TRAD_POOL2)
 		this.config = {
 			input_shape : [40, 100, 1],
+			n_labels : 10,
```

Comment:
IIRC I think `n_labels` should be 12, since we have `unknown` and `silence` on top of the existing ten keywords -- did you account for those additional two elsewhere in the code?

Reply:
oh this config is temporary as mentioned in the comment. Eventually we want this config passed in as part of constructor argument

Reply:
Cool.